sp-validation/creator-node/healthChecks.js

```diff
@@ -0,0 +1,153 @@
+const axios = require('axios')
+const Web3 = require('web3')
+const web3 = new Web3()
+const { promisify } = require('util')
+
+const crypto = require('crypto')
+const assert = require('assert')
+
+const PRIVATE_KEY = process.env.delegatePrivateKey
+const CREATOR_NODE_ENDPOINT = process.env.creatorNodeEndpoint
+
+/**
+ * Parses the environment variables and command line args
+ * export creatorNodeEndpoint=http://creatornode.domain.com
+ * export delegatePrivateKey=f0b743ce8adb7938f1212f188347a63...
+ * NOTE: DO NOT PREFIX PRIVATE KEY WITH 0x
+ */
+function parseEnvVarsAndArgs () {
+  if (!CREATOR_NODE_ENDPOINT || !PRIVATE_KEY) {
+    let errorMsg = `creatorNodeEndpoint [${CREATOR_NODE_ENDPOINT}] or delegatePrivateKey [${PRIVATE_KEY}] have not been exported. `
+    errorMsg += "Please export environment variables 'delegatePrivateKey' (without leading 0x) and 'creatorNodeEndpoint' with https."
+    throw new Error(errorMsg)
+  }
+}
```

Comment:
feels like we should just sanitize the pk input if 0x prefix: `if (pkey.length == 66 && pkey.startsWith('0x')) { pkey = pkey.slice(2,) }` (that is runnable code)
this seems like an unnecessary source of confusion wdyt

can also check endpoint ofc `if (!endpoint.startsWith('https')) { throw new Error('...') }`